src/makeCalculator.js

```diff
@@ -37,7 +37,42 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+
+    result: 0,
+
+    add(a, b) {
+      return a + b;
+    },
+
+    subtract(a, b) {
+      return a - b;
+    },
+
+    divide(a, b) {
+      if (b === 0) {
+        throw new Error('Error');
```

Comment:
Try to describe an error in more detailed way
```suggestion
        throw new Error('Division by 0');
```